app/models/batch_context.rb

```diff
@@ -45,13 +51,17 @@ class BatchContext < ApplicationRecord
   # return [PreAssembly::Batch]
   def batch
     @batch ||= if using_file_manifest
-                 manifest = PreAssembly::FileManifest.new(csv_filename: file_manifest, staging_location: staging_location)
-                 PreAssembly::Batch.new(self, file_manifest: manifest)
+                 PreAssembly::Batch.new(self, file_manifest: file_manifest)
```

Comment:
what do you think of the file_manifest method returning nil instead of this if block here?